blurb

```diff
@@ -650,24 +650,36 @@ Run unit tests.  Only works inside source repo, not when installed.
     print(tests_run, "tests passed.")
 
 
+def find_editor():
+    for var in 'GIT_EDITOR', 'EDITOR':
+        editor = os.environ.get(var)
+        if editor is not None:
+            return editor
+    if sys.platform == 'win32':
+        fallbacks = ['notepad.exe']
+    else:
+        fallbacks = [
+            '/etc/alternatives/editor',
+            '/usr/bin/editor',
+            'editor',
+            'nano',
+        ]
+    for fallback in fallbacks:
+        if os.path.isfile(fallback):
```

Comment:
This will pick up a file in the local directory that happens to be called "nano".